optimize/components/userguide/instant-preview-dashboards.md

```diff
@@ -8,7 +8,7 @@ Camunda Optimize is a comprehensive process optimization tool that helps busines
 and improve efficiency. One of the standout features of Optimize is its ability to automatically generate
 dashboards for each process, providing users with clear insights into process performance.
 
-When a process is imported into Optimize, it automatically generates a dashboard for that process.
+Optimize imports each process deployed from the Modeler and automatically creates a dashboard.
```

Comment:
Technically processes can also be deployed through other means than from the modeler, like through the API? Is it better to write "deployed to Camunda" or sth like that? Do you know any umbrella term that encompasses the C7/C8 worlds? Otherwise we can also write "deployed to platform" or something in those lines, wdyt?

Reply:
Good catch - Camunda Platform it should be :-)